db/migrate/20170208000617_create_registered_users.rb

```diff
@@ -0,0 +1,9 @@
+class CreateRegisteredUsers < ActiveRecord::Migration[5.0]
+  def change
+    create_table :registered_users do |t|
+      t.text :api_key
+      t.integer :status
```

Comment:
I think you're right. I think our API key for external use will be a key that we give users to access our entire api. I don't think it's at the user level. Also, if we need it at the user level we can just add it later. I'll remove it and resubmit the PR.